src/t8_geometry/t8_geometry.cxx

```diff
@@ -253,14 +253,14 @@ t8_geom_handler_get_num_geometries (const t8_geometry_handler_t *geom_handler)
   return geom_handler->registered_geometries.elem_count;
 }
 
-const t8_geometry_c *
+t8_geometry_c *
```

Comment:
Interesting. Why remove `const` qualifier?

Reply:
I put it back .